python/tests/test_drawing.py

```diff
@@ -1913,6 +1919,32 @@ def test_known_svg_ts(self):
             expected_svg = file.read()
         self.assertXmlEquivalentOutputs(svg, expected_svg)
 
+    def test_known_svg_internal_sample_ts(self):
+        n = 5
+        ts = msprime.simulate(n, random_seed=10, mutation_rate=0.8)
+        assert ts.num_mutations > 0
+        tables = ts.dump_tables()
+        nodes = tables.nodes
+        flags = nodes.flags
+        # Set a mixture of internal and leaf samples.
+        flags[:] = 0
+        flags[n // 2 : n + n // 2] = tskit.NODE_IS_SAMPLE
+        nodes.flags = flags
+        ts = tables.tree_sequence()
+        svg = ts.draw_svg(
+            root_svg_attributes={"id": "123"},
+            style=".leaf .sym {fill: green} .sample .sym {fill: cyan}",
+        )
+        # Prettify the SVG code for easy inspection
+        svg = xml.dom.minidom.parseString(svg).toprettyxml()
+        svg_fn = os.path.join(
```

Comment:
pathlib is a bit nicer for this sort of thing: https://docs.python.org/3.7/library/pathlib.html